### PR TITLE
Fix/file input required is not set page 2217

### PR DESCRIPTION
**Pull request by ingmar-stipriaan**

### CODE REVIEW

File: src/components/actionJSButton.js [hunk 1]
```diff
@@ -134,7 +129,7 @@
     };
 
     const buttonProps = {
-      disabled: disabled || isLoading,
+      disabled: disabled || isLoading || loading,
```
Comment (stefan-betty) [inline src/components/actionJSButton.js:132]: You can use `showIndicator` here.

File: src/components/actionJSButton.js [hunk 2]
```diff
@@ -124,7 +119,7 @@
       return undefined;
     };
 
-    const showIndicator = isLoading;
+    const showIndicator = isLoading || loading;
```
Comment (stefan-betty) [inline src/components/actionJSButton.js:122]: Why do we have two loading variables?

Comment (ingmar-stipriaan) [reply src/components/actionJSButton.js:122]: isLoading can be used by the builder to set te loading state. Loading will be returned from  useActionJS when the action is running, but doesn't set the state and useActionJS doesn't have an onLoading option to set te state either.

Comment (stefan-betty) [reply src/components/actionJSButton.js:122]: I think we should change the name of `loading` to `isActionLoading` to make it clear what we are waiting for!

Comment (ingmar-stipriaan) [reply src/components/actionJSButton.js]: Changed it to isLoadingAction

File: src/components/actionJSButton.js (hunk 1 above)
Comment (ingmar-stipriaan) [reply src/components/actionJSButton.js]: I used showIndicator now

**Review by jrquak — APPROVED** [review]

**Pull request merged**